src/libs/actions/App.ts

```diff
@@ -254,6 +255,7 @@ function getOnyxDataForOpenOrReconnect(isOpenApp = false): OnyxData {
  */
 function openApp() {
     getPolicyParamsForOpenOrReconnect().then((policyParams: PolicyParamsForOpenOrReconnect) => {
+        Log.info('[openApp] getPolicyParamsForOpenOrReconnect resolved: ', true, policyParams);
         const params: OpenAppParams = {enablePriorityModeFilter: true, ...policyParams};
 
         API.write(WRITE_COMMANDS.OPEN_APP, params, getOnyxDataForOpenOrReconnect(true));
```

Comment:
My initial experience of this bug was that it _looked like_ `API.write()` did get called, but we end up just stuck in the queue so the request never happens. It feels like maybe some kind of network request is happening that causes the queue to get blocked.